src/Kernel32.Tests.Shared/NTStatusExceptionFacts.cs

```diff
@@ -54,10 +58,11 @@ public void NTStatusException_Informational_Message()
     }
 
     [Fact]
+    [UseCulture("en-US")]
```

Comment:
Well, it seems that all .NET related errors do get translated to English when this attribute is set. It solved all issues that did not call the native `FormatMessage` API. These methods do need the 0x409. We could change the functions that use FormatMessage inside to detected the current culture and call it using the correct code. This would make all the methods with the new `dwlanguage` parameter not change its signature and we could just use this Attribute to solve the problems.